### PR TITLE
feature/Add secret type

### PR DESCRIPTION
**Pull request by oussamaessaji**

### CODE REVIEW

**Review by deadly-panda — APPROVED** [review]
LGTM.
However, I would've prepared the first part of the technical details (secret+type), then added the where it was found detail.

**Review by amine3 — APPROVED** [review]

**Review by PiranhaSa — APPROVED** [review]

**Review by rabsondev — APPROVED** [review]

File: tests/process_input_test.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-"""Tests for the process_input Module."""
+"""Tests for the input_type_handler Module."""
```
Comment (3asm) [inline tests/process_input_test.py:1]: FIle name needs to be changed.

File: agent/trufflehog_agent.py [hunk 2]
```diff
@@ -42,11 +43,16 @@ def _report_vulnz(self, vulnz: list[dict[str, Any]], message: m.Message) -> None
                 continue
             secret_token = utils.escape_backtick(secret_token)
             logger.info("Secret found : %s.", vuln["Redacted"])
+            technical_detail = f"""Secret `{secret_token}` """
+            secret_type = vuln.get("DetectorName")
+            if secret_type is not None:
+                technical_detail += f"""of type `{secret_type}` """
+            technical_detail += f"""found in file `{message.data.get("path")}`"""
```
Comment (3asm) [inline agent/trufflehog_agent.py:50]:
```suggestion
            technical_detail += f"""found in file `{message.data.get("path")}`."""
```

File: agent/trufflehog_agent.py [hunk 3]
```diff
@@ -42,11 +43,16 @@ def _report_vulnz(self, vulnz: list[dict[str, Any]], message: m.Message) -> None
                 continue
             secret_token = utils.escape_backtick(secret_token)
             logger.info("Secret found : %s.", vuln["Redacted"])
+            technical_detail = f"""Secret `{secret_token}` """
+            secret_type = vuln.get("DetectorName")
+            if secret_type is not None:
```
Comment (3asm) [inline agent/trufflehog_agent.py:48]: Is it possible not to have a secret type?

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**